clang/test/SemaSYCL/sycl-empty-header.cpp

```diff
@@ -0,0 +1,10 @@
+// Test that at least an empty integration header file is created when
+// compiling source files with no device sycl construct.
+
+// RUN: mkdir -p %t_dir
```

Comment:
Can we use %T instead?
Otherwise this directory must be deleted manually as well, right?

Reply:
@bader, the https://llvm.org/docs/TestingGuide.html#substitutions seems to indicate that %T is deprecated.
I will modify the test to remove the directory manually.